java/arcs/core/data/util/ReferencablePrimitive.kt

```diff
@@ -31,7 +31,8 @@ data class ReferencablePrimitive<T>(
     val valueRepr: String = value.toString()
```

Comment:
probably out of scope for this pr, but I just noticed: if the value is eg a very long string, does this mean we keep there copies of it: in value, in valueRepr, and in id?

Reply:
I think yes, that's also why we can see lots of debug messages significantly slowing down performance when debug log is enabled by "adb shell setprop log.tag.arcs DEBUG"